drivers/gpu/drm/vc4/vc4_hdmi_regs.h

```diff
@@ -33,11 +33,12 @@ enum vc4_hdmi_field {
 	HDMI_CEC_CNTRL_3,
```

Comment:
The commit message is in too long a line (72 char limit), and includes two typos - either go with "Fixes an incorrect register's address" or "Fixes an incorrect register address".